pages/integrations/inputs/okta_input.rst

```diff
@@ -0,0 +1,32 @@
+.. _okta_input:
+
+*********************
+Okta Log Events Input
+*********************
+
+.. attention:: This is an Enterprise Integrations feature and is only available since Graylog version 3.3 and thus requires an Enterprise license.
+
+
+This input allows Graylog to pull Okta Log Events from your okta domain.
+
+For this input plugin to function as expected, the following items need to be supplied in the input module.
+
+1) Domain name
+    The domain name associated with the IP address of the inbound event request (where the Okta Log Events are located).
```

Comment:
```suggestion
    Your Okta Domain (also known as your Okta URL).  You can copy your domain from the Okta Developer Console.
```

Reply:
Also consider adding a link to https://developer.okta.com/docs/guides/find-your-domain/overview/